armi/tests/test_plugins.py

```diff
@@ -42,13 +43,25 @@ def defineFlags():
         return {"SUPER_FLAG": utils.flags.auto()}
 
 
+class SillyAxialExpansionChanger(AxialExpansionChanger):
+    pass
+
+
+class SillyAxialPlugin(plugins.ArmiPlugin):
+    @staticmethod
+    @plugins.HOOKIMPL
+    def getAxialExpansionChanger() -> type[SillyAxialExpansionChanger]:
+        return SillyAxialExpansionChanger
+
```

Comment:
Can you add some comments to the above for posterity? I could totally see myself coming back here in 6+ months and being like "what the heck is this again....?" 

E.g., within `SillyAxialExpansionChanger` something like "a test subclass that an ARMI Application could create"

and then within `SillyAxialPlugin` something like "the plugin hook that an ARMI Application would have to register the `SillyAxialExpansionChanger` subclass"

Reply:
Sure!

Reply:
b1270c93